index.js

```diff
@@ -11,11 +12,44 @@ server.get('/payments', (req, res) => {
     res.status(200).send(payments);
 });
 
+server.get('/payments/:id', (req, res) => {
+    const payment = payments.find( payment => payment.id === req.params.id);
+    if(payment){
+        res.status(200).send(payment);
+        return;
+    }
+    res.status(404).send({message: 'Wrong id'});
+});
+
+server.post('/payments', (req, res) => {
+    const payment = {...req.body};
+    Object.assign(payment, {
+        id: uuidv4(),
+        status: 'Pending'
+    });
+    payments.push(payment);
+    res.status(200).send(payment);
+});
 
+server.put('/payments/:id', (req, res) => {
+    const payment = payments.find( payment => payment.id === req.params.id);
+    if(!payment || !req.body.description){
```

Comment:
hi 
well, not really, coz:
- in case of `!payment` we should send status code 404 NOT_FOUND
- in case of  `!req.body.description` we fall into another error: 400 BAD_REQUEST
please, treat them differently.